pkg/specgen/generate/pod_create.go

```diff
@@ -30,19 +30,16 @@ func buildPauseImage(rt *libpod.Runtime, rtConfig *config.Config) (string, error
 		return imageName, nil
 	}
 
-	// NOTE: Having the pause binary in its own directory keeps the door
-	// open for replacing the image building with using an overlay root FS.
-	// The latter turned out to be complex and error prone (see #11956) but
-	// we may be able to come up with a proper solution at a later point in
-	// time.
-	pausePath, err := rtConfig.FindHelperBinary("pause/pause", false)
+	// Also look into the path as some distributions install catatonit in
+	// /usr/bin.
+	catatonitPath, err := rtConfig.FindHelperBinary("catatonit", true)
```

Comment:
I don't see how this can work unless `containers.conf` is updated to include `/usr/libexec/catatonit` in `helper_binaries_dir`. Is that in progress (presumably elsewhere)?

Reply:
It should work, at least on fedora `/usr/libexec/podman/catatonit` is a symlink to the actual catatonit path.
```
$ ll /usr/libexec/podman/catatonit 
lrwxrwxrwx. 1 root root 32 Sep 16 15:40 /usr/libexec/podman/catatonit -> /usr/libexec/catatonit/catatonit
```

Reply:
Oops - thank you, I had missed that.